.github/workflows/knative-docs-image.yaml

```
@@ -5,6 +5,8 @@ on:
     branches: ['main']
     paths:
     - "code-samples/serving/hello-world/helloworld-go/**"
+    - "code-samples/eventing/bookstore-sample-app/solution/frontend/**"
+    - "code-samples/eventing/bookstore-sample-app/solution/node-server/**"
```

Would this be simpler? not really sure how this workflow works
```suggestion
    - "code-samples/eventing/bookstore-sample-app/**"
```